src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorWrapper.java

```diff
@@ -266,6 +268,8 @@ public RawPacket reverseTransform(RawPacket pkt)
                 pkt.getPayloadLength(), pkt.getSSRCAsLong());
         }
 
+        // FIXME what if we don't have abs-send-time header extension.
```

Comment:
There is code just above which waits for 30 such packets and switches back to the SS impl